dataplane/linux/int_dataplane.go

```diff
@@ -301,28 +321,39 @@ func NewIntDataplaneDriver(config Config) *InternalDataplane {
 		config.RulesConfig.IptablesMarkNonCaliEndpoint)
 
 	// Auto-detect host MTU.
-	if mtu, err := findHostMTU(config.MTUIfacePattern); err != nil {
+	hostMTU, err := findHostMTU(config.MTUIfacePattern)
+	if err != nil {
 		log.WithError(err).Fatal("Unable to detect host MTU, shutting down")
-	} else {
```

Comment:
I refactored this a little to make it easier to read. Let me know what you think.

Reply:
Is it intentional that the code that _was_ in the `else` block will now run even when there is an error?

Reply:
The Fatal log will exit, but maybe it's clearer what's going on if the code is back in a else block?

Reply:
ahh I see, I'm not familiar with the fatal flag :) instead of putting back in the else-block, maybe a gratuitous `return`? Only a small thing anyway, my main worry was that it would continue executing, but since it doesnt, my worries are gone!

Reply:
Was thinking the same about a gratuitous `return`, I'll add that in!